articles/anomaly-detection/passwords-checking.md

```diff
@@ -0,0 +1,229 @@
+Bulk Passwords Checking
+=======================
+
+In Auth0 we focus on bring you the tools you need to keep your systems and your users safe.
```

Comment:
At Auth0, we focus on bringing you the tools you need to keep your systems and your users safe.